src/services/shell.cpp

```diff
@@ -17,6 +17,8 @@
 #include <cloysterhpc/NFS.h>
 #include <cloysterhpc/cluster.h>
 #include <cloysterhpc/repos.h>
+#include <cloysterhpc/runner.h>
+#include <sys/socket.h>
```

Comment:
Do we need `socket.h`?

Reply:
Probably not. I think the IDE added this automatically and I forgot to remove

Reply:
```suggestion
```